README.md

```diff
@@ -173,6 +173,20 @@ Possible values of this variable:
 * `{state: absent}`: the ansible host name is unset in the insights-client config file and Host Based Inventory (HBI) is updated to use the system host name.
 * any other string value: the ansible host name is changed in Host Based Inventory (HBI).
 
+```yaml
+    rhc_insights:
+      display_name: "Example Host"
+```
+
+Configures the display name with a custom value for the system record in Host Based Inventory (HBI). This display name is only used to identify the host in the Invenotry.
```

Comment:
typo "Invenotry"

Reply:
If the only comment on the description is a typo, it’s good news! 🥳